packages/fx-core/src/client/graphClient.ts

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { hooks } from "@feathersjs/hooks";
+import { M365TokenProvider, LogProvider } from "@microsoft/teamsfx-api";
+import { AxiosInstance } from "axios";
+import { ErrorContextMW } from "../common/globalVars";
+import { GetTeamsAppSettingsResponse } from "./interfaces/GetTeamsAppSettingsResponse";
+import {
+  GraphTeamsAppSettingsReadScopes,
+  GraphTeamsChannelCreateScopes,
+  GraphTeamsChannelReadScopes,
+  GraphTeamsInstallAppScopes,
+  GraphTeamsTeamCreateScopes,
+  GraphTeamsTeamReadScopes,
+} from "../common/constants";
+import { GetJoinedTeamsResponse } from "./interfaces/GetJoinedTeamsResponse";
+import { GetChannelResponse } from "./interfaces/GetChannelResponse";
+import { WrappedAxiosClient } from "../common/wrappedAxiosClient";
+import { CreateChannelResponse } from "./interfaces/CreateChannelResponse";
+import { CreateTeamAndChannelResponse } from "./interfaces/CreateTeamAndChannelResponse";
+import { waitSeconds } from "../common/utils";
+import { getLocalizedString } from "../common/localizeUtils";
+
+export class GraphClient {
```

Comment:
We already have a GraphAPIClient, it looks confusing for another GraphClient. Can we unify into one?